goi18n/testdata/expected/R.go

```diff
@@ -0,0 +1,26 @@
+// DON'T CHANGE THIS FILE MANUALLY
+// This file was generate using the command:
+// $ goi18n constants
+
+package R
+
+// DDays id:d_days
```

Comment:
This comment doesn't seem very useful to me since it just repeats what is in the line of code below. I would remove it.